tox.ini

```diff
@@ -19,6 +19,12 @@ deps = {[testenv:py33]deps}
 [testenv:py36]
 deps = {[testenv:py33]deps}
 
+[testenv:py37]
```

Comment:
We should remove py33 and py34 from tox.ini (we'll have to copy py33's `deps` to py35, and then update references to `py33` to be `py35`).

Reply:
Done

Reply:
When reviewing `tox.ini`, I now added PyPy3 to the list for coherence